styles.css

```diff
@@ -1,3 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap");
+
 body {
-  background-color: white;
+  margin: 0;
+  min-height: 100vh;
+  font-family: "Open Sans", sans-serif;
+  background-image: linear-gradient(90deg, #fcecfe, #fbf6e7, #e6fcf5);
+}
+
+.content-container {
+  margin: 0 auto;
+}
+.header {
+  text-align: center;
+  color: black;
+  font-style: italic;
+}
+.navigation-bar {
+  position: fixed;
+  width: 100%;
+}
+
+.navigation-bar ul {
+  margin: 0;
+  background-color: #e6fcf5;
+  padding: 0 16px;
+  flex-wrap: wrap;
+  list-style-type: none;
+  display: flex;
+  gap: 32px;
+  justify-content: flex-end;
+  min-height: 60px;
+  align-items: center;
+}
+
+.navigation-bar a {
+  text-decoration: none;
+  color: black;
+}
+
+.navigation-bar a:hover {
+  text-decoration: underline;
+}
+
+.content-container {
+  display: flex;
+  place-items: center;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.typing {
+  width: 20ch;
+  animation: typing 2s steps(20), blink 0.5s step-end infinite alternate;
+  white-space: nowrap;
+  overflow: hidden;
+  border-right: 1px solid;
+  font-family: monospace;
+  font-size: 1em;
+  margin: 0 auto;
+}
+
+@keyframes typing {
+  from {
+    width: 0;
+  }
+}
+
+@keyframes blink {
+  50% {
+    border-color: transparent;
+  }
+}
+
+#about {
```

Comment:
when i clicked on the "about" button on the navigation bar, the alignment seems a little off - didn't show the content of the "about" section. is it because of the size of the "about" section?

Reply:
actually, it's because your nav bar has `display: fixed` which is causing it to cover the section. 

one workaround, without removing the fixed attribute, would be to give the sections a bit more height and vertically align the text so they're not obscured by the navbar.

the more difficult workaround would be to fade out your nav bar as the user scrolls down. that requires a bit more css magic. maybe something like this: https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_navbar_hide_scroll